rbc/src/main/java/com/library/rbc/controller/BookController.java

```diff
@@ -48,4 +52,10 @@ public Page<BookDto> returnBooksBy(@ParameterObject @PageableDefault(size = 12)
       @RequestParam(required = false) List<String> bookStatuses) {
     return bookService.getBooksBy(pageable, bookCategories, bookStatuses);
   }
+
+  @PostMapping("/upload-image/{bookId}")
```

Comment:
I just noticed this in the other PR:
Endpoint suggestion: `@PostMapping("/{bookId}/image")`